projects/fsastorefrontlib/src/assets/translations/en/quote.en.ts

```diff
@@ -29,5 +29,75 @@ export const quote = {
       ERROR: 'Error',
       na: 'N/A',
     },
+    quoteSummary: 'Quote Summary',
```

Comment:
Since we have already majority of these translations, can we perhaps unite them in one file`insurance-products` or something like that?

Reply:
So you mean to have all translations for (policy, quote, claim) pages into one for example insurance-products file?

Reply:
Yup, check with others just in case, but I don't see why we shouldn't do that. The way it is now, we have a lot of duplicated code.

Reply:
If everyone agrees we can maybe create improvement ticket for this, so we can cover all insurance products (policy, quote, claim) and carefully change all those references to those translations

Reply:
Everyone agrees... Please create the ticket 😈